test/models/comment_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class CommentTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
Kindly add unit tests for the `User`, `Post`, `Comment`, and `Like` models. You currently do not have any tests for your model validations.

* Model specs **Validation** tests Example:

  ```
  describe 'validations' do
    it { should validate_presence_of(:name) }
  end
  ```